test/chromedriver-specs.js

```diff
@@ -28,12 +27,15 @@ function nextError (cd) {
 }
 
 async function assertNoRunningChromedrivers () {
-  // let res = await Q.nfcall(psNode.lookup, {command: /([^ ]*)chromedriver$/,
-  //                                          psargs: 'aux'});
-  let f = B.promisify(psNode.lookup);
-  let res = await f({command: /([^ ]*)chromedriver$/, psargs: 'aux'});
+  let {stdout} = await exec('ps', ['aux']);
```

Comment:
It does not. It's also just a development dependency. We used it in order to assert that no chromedrivers were running. So getting rid of it should not affect the actual working on the driver on any system.